docs/content/preview/yugabyte-platform/back-up-restore-universes/configure-backup-storage.md

```diff
@@ -102,23 +102,53 @@ You can configure Google Cloud Storage (GCS) as your backup target, as follows:
 
 1. Navigate to **Configs > Backup > Google Cloud Storage**.
 
-2. Click **Create GCS Backup** to access the configuration form shown in the following illustration:
+1. Click **Create GCS Backup** to access the configuration form shown in the following illustration:
 
-    ![GCS Configuration](/images/yp/cloud-provider-configuration-backup-gcs.png)
+    ![GCS Configuration](/images/yp/cloud-provider-configuration-backup-gcs-stable.png)
 
-3. Use the **Configuration Name** field to provide a meaningful name for your backup configuration.
+1. Use the **Configuration Name** field to provide a meaningful name for your backup configuration.
+
+1. Enter the URI of your GCS bucket in the **GCS Bucket** field. For example, `gs://gcp-bucket/test_backups`.
+
+1. Select **Use GCP IAM** if you're using [GKE service account](#gke-service-account-based-iam-gcp-iam) for backup and restore.
 
-4. Complete the **GCS Bucket** and **GCS Credentials** fields.
+1. Complete the **GCS Credentials** field.
```

Comment:
Super unhelpful!

Reply:
I assumed stable had the latest copy. Will revert them in both preview and stable.